content/en/apps/tutorials/local-setup.md

```diff
@@ -160,7 +160,8 @@ Then deploy the blank project onto your local test environment with the command:
 cht --url=https://medic:password@localhost --accept-self-signed-certs
 ```
 
-If the above command shows an error similar to this one `ERROR Error: Webpack warnings when building contact-summary` you will need to install all the dependencies and libraries it needs, then you need to restart the docker-compose and try again.
+If the above command shows an error similar to this one `ERROR Error: Webpack warnings when building contact-summary` you will need to install all the dependencies and libraries it needs, then you need to restart the docker-compose and try again. Run the following to install the dependancies.
+If this error is persistent, please create a new directory and initialise a new project in the new location (this location should not be in your cht-core folder).
```

Comment:
I'm tempted to say that we should make this a code check in cht-conf, but this change really just adds detail to the above line that's already here. Given that, it seems fine for now.

Slight wording suggestion:
```suggestion
If this error is persistent, please create a new directory and initialise a new project in the new location (outside of your `cht-core` directory tree).
```

I'll approve this now to unblock. If you like your original wording more, it's ok to ignore my suggestion and go with whatever you prefer.